src/mumble/Plugins.cpp

```diff
@@ -409,6 +398,21 @@ bool Plugins::fetch() {
 	return bValid;
 }
 
+static MumblePIDLookupStatus lookup_func(MumblePIDLookupContext ctx, const wchar_t *procname, unsigned long long int *pid) {
+
+	std::multimap<std::wstring, unsigned long long int> *pids = reinterpret_cast<std::multimap<std::wstring, unsigned long long int> *>(ctx);
```

Comment:
I assume this is a multimap for a reason instead of just a map (probably multiple instances of the same exe would have the same keys?). However this lookup function gives me no way to query anything but one the entries for the given key.

Reply:
This is the method we always used.

It's a `multimap` because we use the process name as key.
Multiple instances result in multiple keys with the same value.

Reply:
Which was my whole point. We have a multimap but the lookup function can never make use of the "multi" part of it. I'm not sure this was intended. The design with context and the return values looks like maybe this function wasn't fully finished.

Reply:
Probable.